src/renderer/hooks/useMutationObserver.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from "react";
+
+const config: MutationObserverInit = {
+  subtree: true,
+  childList: true,
+  attributes: false,
+  characterData: false
+};
+
+export function useMutationObserver(
```

Comment:
is this instead of `mobx`?

Reply:
Not really sure if "instead" is a proper word here. This is just new react hook which can be used in any functional component. It doesn't relate to mobx at all.

Reply:
Yep, nothing related to `mobx` here. It's just a`react` hook with https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver API on board.